### ISSUE TITLE
SENSEI does not work with ParaView 5.10

### ISSUE THREAD

**Issue by mvictoras** [SENSEI-insitu/SENSEI#37]
SENSEI does not compile when linking against PV 5.10.

```
CMake Error at /lus/grand/projects/visualization/mvictoras/argo/install/paraview/v5.10.0/lib64/cmake/paraview-5.10/paraview-config.cmake:167 (find_package):
  Found package configuration file:

    /lus/grand/projects/visualization/mvictoras/argo/install/paraview/v5.10.0/lib64/cmake/paraview-5.10/vtk/vtk-config.cmake

  but it set VTK_FOUND to FALSE so package "VTK" is considered to be NOT
  FOUND.  Reason given by package:

  Could not find the VTK package with the following required components:
  AcceleratorsVTKmFilters, FiltersOpenTURNS, RenderingRayTracing, m.

Call Stack (most recent call first):
  CMake/catalyst.cmake:26 (find_package)
  CMakeLists.txt:20 (include)
```

**Comment by c-wetterer-nelson** [comment]
I've bumped into this issue as well. If you compile your ParaView with ALL those libraries (it's not necessarily obvious what options correspond with which module in the paraview cmake lists) then you can get it to link. However, I definitely think SENSEI does not need all or even any of those listed modules to operate (unless specifically requested). 

@burlen 

Does the Histogram miniapp use the FiltersOpenTURNS stuff perchance? 

The 'm' refers to VTKm I believe. Unless you're requesting it, that should not be required.

RenderingRayTracing is a weird one to require. Is there specific need for ray tracing in vanilla SENSEI? Looking through the sensei build options, I'm not sure what's including this.

**Comment by burlen** [comment]
@c-wetterer-nelson I think porting SENSEI to use PV 5.10 is a kitware related task (@patrickoleary do you agree?). In that case you should take a look at it and resolve the issues, and while your at it, please add a version check to `find_package(ParaView VERSION X)` so that it's clear to users which version of PV we target. If you guys aren't going to port to PV 5.10 then we can just add the version check and fix on PV 5.9.

**Comment by burlen** [comment]
@mvictoras until we port SENSEI to PV 5.10, you should make use of PV 5.9 (known to work). If having PV 5.10 is a show stopper then please bring it up at the  next meeting (or mention here if it's more urgent than that)
